src/latexify/config.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+import dataclasses
+
+from typing import Any
+
+
+@dataclasses.dataclass(frozen=True)
+class Config:
+    identifiers: dict[str, str] | None
+    expanded_functions: set[str] | None
+    reduce_assignments: bool
+    use_math_symbols: bool
+    use_raw_function_name: bool
+    use_signature: bool
+    use_set_symbols: bool
+
+    def merge(self, *, config: Config | None = None, **kwargs) -> Config:
```

Comment:
Add docstring for all public functions. It is required to follow the Google's style: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods